locales/en.json

```diff
@@ -301,6 +301,17 @@
     "log_tools_upgrade": "Upgrade system packages",
     "log_tools_shutdown": "Shutdown your server",
     "log_tools_reboot": "Reboot your server",
+    "log_admin_bash_history": "History of commands ran by admin",
+    "log_alternatives": "Change to default programs",
+    "log_auth": "Connection history",
+    "log_btmp": "Failed login attempts",
+    "log_dpkg": "History of installed debian packages",
+    "log_fail2ban": "Banning history",
+    "log_history": "History of installed debian packages with apt",
```

Comment:
Well uh, is this for apt, or is this the category for the admin/root history ..? Shouldn't this be `log_package` ..?

Also I would really rename the `package` category to `apt_and_dpkg` or just `apt` or `system_packages` to make a clear distinction to apps or other packages